sky/engine/core/dart/painting.dart

```diff
@@ -165,7 +165,8 @@ class Path extends NativeFieldWrapperClass2 {
   /// Adds a new subpath that consists of the straight lines and
   /// curves needed to form the rounded rectangle described by the
   /// argument.
-  void addRRect(RRect rrect) native "Path_addRRect";
+  void addRRect(RRect rrect) => _addRRect(rrect._value);
+  void _addRRect(Float32List rrect) native "Path_addRRect";
```

Comment:
are you using Float32 for consistency, because skia expects floats not doubles, because it's faster, or some other reason?